fabric/Makefile

```diff
@@ -11,8 +11,24 @@ FABRIC_PEER_BIN = $(FABRIC_PATH)/build/bin/peer
 FABRIC_PATCHED = $(FABRIC_PATH)/.fpc_patched
 FABRIC_CURRENT_DIFF_COMMITS_CMD = git log $(FABRIC_VERSION_TAG)..HEAD --oneline
 
-build: native
+FABS = _internal
```

Comment:
I guess here we should move `bin` to `scripts`, and `_internal` to `bin`.

Reply:
mh, I was thinking the same. But then decided that for me it makes more sense when we provide our peer wrapper in `fabric/bin` as this is something we give to the FPC users.

It seems that current 'scripts' only contain helper for testing and ci. That said, it would make sense for me to move `ledger_*` scripts to `scripts`. The other wrapper files (configtxgen and orderer) are also not really needed. 

In testing, can't we just define leverage the PATH to run the peer, orderer, and configtxgen commands?

We could say if FPC is used: do something like `PATH=$FPC_PATH/fabric/bin:$FPC_PATH/fabric/_internal/bin:$FPC`